### DISCUSSION TITLE
Google Sheets Case Study

### DISCUSSION THREAD

**Discussion by gkdn** [google/j2cl#247] [General]
https://web.dev/case-studies/google-sheets-wasmgc

**Comment by niloc132** [comment]
Thanks for sharing this! 

From https://github.com/google/j2cl/blob/master/docs/getting-started-j2wasm.md, it appears that experimental chrome features are required to use j2wasm:
> NOTE: J2CL currently uses experimental Wasm String functionality which might not be enabled yet. 

Is that comment out of date, or is there another way around this issue that J2CL uses?

**Reply by gkdn** [reply]
J2CL still relies experimental String functionality however the underlying spec recently reach to phase 3 so should be enabled by default relatively soon:
https://github.com/WebAssembly/js-string-builtins/blob/main/proposals/js-string-builtins/Overview.md

So it requires manually enabling it in Chrome today or an origin trial :
https://chromestatus.com/feature/6695587390423040
